CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 2.50.0-rc.0 / 2024-01-26
+
+* [FEATURE] Add support for ingesting zeros as created timestamps (enabled under the feature-flag `created-timestamp-zero-ingestion`). #12733 #13279
+* [FEATURE] TSDB/{index,compact}: allow using custom postings encoding format. #13242 #13407 #13391
```

Comment:
ok I will remove it. But "user" can mean so many different profiles. Here it could be a developer that would still be interested about this feature don't you think ?